VisualActivityViewController.swift

```diff
@@ -183,8 +183,11 @@ final class VisualActivityViewController: UIActivityViewController {
     override func viewWillDisappear(_ animated: Bool) {
         super.viewWillDisappear(animated)
         
-        UIView.animate(withDuration: fadeOutDuration) {
-            self.preview?.alpha = 0
+        UIView.animate(withDuration: fadeOutDuration,
```

Comment:
I prefer method signatures to only be on newlines if the signature is fairly long. Please put the `animation` parameter name on the same line as the duration parameter.